src/service/index.ts

```diff
@@ -1,91 +1,157 @@
 import { Project, Task, FetchIssuesParams, Profile } from '@/types'
 import http from './instance'
 
-const api = {
-  fetchUserInfo: async (code: string) => {
-    try {
-      const response = await http.get('/auth/github/callback', {
-        params: { code },
-      })
-
-      console.log('User info:', response.data)
-      if (response.data.data && response.data.data.jwt) {
-        return response.data.data // 返回 JWT
-      }
-    } catch (error) {
-      console.error('Error fetching user info:', error)
+export async function fetchUserInfo(code: string) {
+  try {
+    const response = await http.get('/auth/github/callback', {
+      params: { code },
+    })
+
+    console.log('User info:', response.data)
+    if (response.data.data && response.data.data.jwt) {
+      return response.data.data // 返回 JWT
     }
-  },
-  fetchTask: async (githubId: string): Promise<Task | null> => {
-    try {
-      const response = await http.get(`/task/${githubId}`)
+  } catch (error) {
```

Comment:
Do we really need to cath error here? If throw error directly, what will happen?

Reply:
We should handle errors in the central entry method. Let me check.